pkg/asset/installconfig/ovirt/credentials.go

```diff
@@ -1,92 +1,231 @@
 package ovirt
 
 import (
+	"crypto/tls"
+	"crypto/x509"
 	"fmt"
-	"net/url"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strconv"
 
+	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"gopkg.in/AlecAivazis/survey.v1"
 )
 
-func askCredentials() (Config, error) {
-	c := Config{}
+// Add PEM into the System Pool
+func (c *clientHTTP) addTrustBundle(pemFilePath string) error {
+	c.certPool, _ = x509.SystemCertPool()
+	if c.certPool == nil {
+		logrus.Debug("Failed to load cert pool.... Creating new cert pool")
+		c.certPool = x509.NewCertPool()
+	}
+
+	pem, err := ioutil.ReadFile(pemFilePath)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to read the cert: %s", pemFilePath)
```

Comment:
Make sure error does not start with capital letter and end with a full stop. Please make sure this is fixed across this change set I can see this problem in multiple places